src/components/AppHeader.vue

```diff
@@ -43,10 +43,14 @@
 			</a-col>
 			<a-col :span="4">
 				<div>
-					<WalletConnect />
+					<WalletConnect @updateWalletAddress="updateWalletAddress($event)" />
 				</div>
 			</a-col>
 		</a-row>
+		<a-row v-if="walletAddress" justify="end">
+			Address: <br />
+			{{ walletAddress }}
```

Comment:
why breaking it in 2 lines?

Reply:
It's a line break. I will do something like the below. I felt like this would look good.
Address:
<<the address will come here>>